### PR TITLE
Tic tac toe game.

### PR DESCRIPTION
**Pull request by PaulChukhonski**

### CODE REVIEW

File: ticTacToe/ticTacToe.cpp [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+#define MAIN_CASES_NUM '4'
+
+void initializeGameField(char cells[]);
+void initializeNumeratedField(char cells[]);
+void showField(char cells[]);
+void showAllFields(char cells[], char numeratedCells[]);
+string inputPosition(string str, bool isPlayerX);
+bool isCorrectInput(string str, char last);
+int strToInt(string str);
+bool canPlay(char cells[]);
```
Comment (VadimAlkhimenok) [inline ticTacToe/ticTacToe.cpp:22]: use isCanPlay

File: ticTacToe/ticTacToe.cpp [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+		{
+			symbol = checkWinner(symbol, countX, countO, SIDE);	
+		}	
+		
+		countX = 0, countO = 0, countEmpty = 0, pos = -1;
+		end++;
+	}
+		
+	return symbol;
+}
```
Comment (VadimAlkhimenok) [inline ticTacToe/ticTacToe.cpp:430]: It's a very big function

File: ticTacToe/ticTacToe.cpp (hunk 1 above)
Comment (PaulChukhonski) [reply ticTacToe/ticTacToe.cpp]: corrected according to comment

File: ticTacToe/ticTacToe.cpp (hunk 2 above)
Comment (PaulChukhonski) [reply ticTacToe/ticTacToe.cpp]: corrected according to comment

Comment (PaulChukhonski) [reply ticTacToe/ticTacToe.cpp]: fixed according to comment

**Pull request merged**